### PR TITLE
fix: `ng.join` bug #122

### PR DESCRIPTION
**Pull request by CorvusYe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This source code is licensed under Apache 2.0 License.
 -->
 
 # 1.1.2-SNAPSHOT
+- fix: `ng.join` bug [#122](https://github.com/nebula-contrib/ngbatis/issues/122)
```
Comment (wey-gu) [inline CHANGELOG.md:9]: The changing log for the snapshot is really elegant, ❤.

**Review by wey-gu — APPROVED** [review]

**Comment by wey-gu** [comment]
Let's see your activity in GMT +8 from here https://ossinsight.io/analyze/CorvusYe

Take good rests at night!!!

Screenshot 2023-01-12 at 10 18 40

**Comment by CorvusYe** [comment]
Wow, what statistics is this through?

**Comment by wey-gu** [comment]
> https://ossinsight.io/analyze/CorvusYe

your public GitHub activities, see https://ossinsight.io/analyze/CorvusYe , there are many more things could be checked :)

**Comment by CorvusYe** [comment]
Cool !

**Pull request merged**